packages/kbn-discover-utils/src/data_types/logs/logs_context_service.ts

```diff
@@ -28,15 +27,32 @@ export const DEFAULT_ALLOWED_LOGS_BASE_PATTERNS = [
   'winlogbeat',
 ];
 
-export const createLogsContextService = (_deps: LogsContextServiceDeps = {}) => {
-  // This is initially an hard-coded set of well-known base patterns,
-  // we can extend this allowed list with any setting coming from uiSettings
-  const ALLOWED_LOGS_DATA_SOURCES = [createRegExpPatternFrom(DEFAULT_ALLOWED_LOGS_BASE_PATTERNS)];
+export const createLogsContextService = async ({
+  logsDataAccessPlugin,
+}: LogsContextServiceDeps) => {
+  let logSources;
 
+  if (logsDataAccessPlugin) {
```

Comment:
ℹ️ The plugin had to be made optional due to:

`Error: X-Pack plugin or bundle with id "logsDataAccess" is required by OSS plugin "discover", which is prohibited.`